pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -49,13 +49,6 @@ func init() {
 	registerAlgorithmProvider(defaultPredicates(), defaultPriorities())
 
 	// IMPORTANT NOTES for predicate developers:
```

Comment:
I think this line should be removed also.

Reply:
Emm ... this line looks like a valid warning, for the below lines.